src/index.js

```diff
@@ -1,6 +1,22 @@
 import * as babylon from '@babel/parser';
 import * as t from '@babel/types';
 
+const applyFSPropertiesWithRef = 'applyFSPropertiesWithRef';
+const elements = [
```

Comment:
Should we have test cases for all these components? not sure how easy/difficult it would be to create them or what they're value would be.

Reply:
IMO they wouldn't serve too much purpose. The only thing we need to test is that a component in that list would get the rewrite, and a component out of that list would not get it - and we have tests covering that scenario.